src/compas_invocations2/grasshopper.py

```diff
@@ -0,0 +1,191 @@
+"""
+Adapted from: https://github.com/diffCheckOrg/diffCheck/blob/main/invokes/yakerize.py
+
+Yakerize.py was originally developed as part of the DiffCheck plugin by
+Andrea Settimi, Damien Gilliard, Eleni Skevaki, Marirena Kladeftira (IBOIS, CRCL, EPFL) in 2024.
+It is distributed under the MIT License, provided this attribution is retained.
+"""
+
+import os
+import shutil
+import tempfile
+
+import invoke
+import requests
+import toml
+
+from compas_invocations2.console import chdir
+
+YAK_URL = r"https://files.mcneel.com/yak/tools/latest/yak.exe"
+
+
+def _download_yak_executable(target_dir: str):
+    response = requests.get(YAK_URL)
+    if response.status_code != 200:
+        raise ValueError(f"Failed to download the yak.exe from url:{YAK_URL} with error : {response.status_code}")
+
+    target_path = os.path.join(target_dir, "yak.exe")
+    with open(target_path, "wb") as f:
+        f.write(response.content)
+    return target_path
```

Comment:
this will only work on Windows, right?

Reply:
is this mean to only run in a workflow? or also on local machines? the yak executable is locally available... also on mac.